fastapi/params.py

```diff
@@ -93,6 +96,7 @@ def __init__(
             deprecated=deprecated,
             example=example,
             examples=examples,
+            include_in_schema=include_in_schema,
```

Comment:
[discussion] don't think should/could hide path parameter, the path parameter already shown during route declaration. For example:


```python
@app.get('/user/{uid}')
def get_user_info(uid: str = Path(...)):
    pass
```

Though you could hide the validation info of this path parameter, but people who have access to document already know there is a path parameter called `uid`.

Reply:
Agreed @astraldawn, it's fine to leave it as is. I don't see why someone would want to hide the path param, but at the same time if they can hide the others, they should be able to hide this as well.